docs/overloading-wp-behavior.md

```diff
@@ -124,56 +124,114 @@ The `@snapwp/blocks` package provides a large and growing number of [Block Compo
 
 First, identify the block you want to overload. Each block has a unique name (e.g., `CoreParagraph`, `CoreImage`). You can find this name in the block's data when fetched via WPGraphQL.
 
-### 2. Creating a Custom Component
+### 2. Creating a Custom Component for Core Paragraph
```

Comment:
Can we revert this whole section?

The types and real-world implementation turn the example to noise in context ("Why would I overload a paragraph block to display the cookies).

And clearly the block types need more work before we start telling users to consume something generic like `BaseAttributes` and know that means a block component...